testpilot/frontend/static-src/app/views/testpilot-promo-view.js

```diff
@@ -49,8 +49,6 @@ export default BaseView.extend({
   },
 
   installClicked() {
-    installAddon(this, 'ExperimentDetailsPage Interactions', () => {
-      this.parentView.render();
-    });
+    installAddon(this, 'ExperimentDetailsPage Interactions', () => {});
```

Comment:
This actually caused it to render twice, since I already had this in the experiment page view:

```javascript
app.me.on('change:hasAddon', this.render, this);
```
It took me an embarrassing amount of time to figure out why it would quickly dismiss the modal.